libs/core/src/lib/analytics/feature-flag.service.ts

```diff
@@ -47,8 +48,23 @@ export class FeatureFlagService {
         }, {}),
     ),
   );
+  private backendRemoteFeatures: Observable<Features> = combineLatest([this.featuresData, this.accountMd5]).pipe(
+    map(([data, md5]) =>
+      Object.entries(data)
+        .filter(([key]) => !key.startsWith(FEATURE_PREFIX))
+        .reduce((acc, [key, feature]) => {
+          return {
+            ...acc,
+            [key]: feature?.rollout === 100 || (feature?.variants?.account_id_md5 || []).includes(md5 || ''),
+          };
+        }, {}),
+    ),
+  );
   private stageFeatures = new BehaviorSubject<Features>({ ...stageFeatures, ...this.getCustomFeatures() });
-  private features: Observable<Features> = this.remoteFeatures.pipe(
+  private applicationFeatures: Observable<Features> = this.applicationRemoteFeatures.pipe(
+    map((features) => ({ ...features, ...this.getCustomFeatures() })),
```

Comment:
I guess this line is not needed, because at the moment backend flags are not customizable with the konami code

Reply:
ok, I wasn't sure what those customFeatures were about. Let me clean that up